code/modules/events/spacevine.dm

```diff
@@ -557,3 +557,7 @@
 	name = "dense space vines"
 	opacity = TRUE
 	icon_state = "Hvy1"
+
+/obj/structure/spacevine/weak
+    name = "weak space vines"
+    max_integrity = 25
```

Comment:
could you add a description to this subtype? Not required, but it's far fluffier than just calling the thing weak vine.